scripts/commands.js

```diff
@@ -18,6 +20,17 @@ exports.study = async (message, args, guildRoles) => {
     await message.delete()
 }
 
+exports.stop = async (message, args, guildRoles) => {
+    guildRoles.forEach(r => {
+        r.role.name === 'Studying' ? message.member.roles.remove(r.role.id) : ''
+    })
+
+    await message.author.send('Study Mode Ended!')
```

Comment:
Do you need this await? Would it be better to do this asynchronously?

Reply:
In that case, you need the `await` on the `message.member.roles.remove` because that's the bit that you need to wait for to ensure it is done, not sending the message

Reply:
Thank you - consider it done 😎